src/main/kotlin/com/beust/kobalt/maven/PomGenerator.kt

```diff
@@ -55,7 +52,7 @@ public class PomGenerator @Inject constructor(@Assisted val project: Project) {
 
         val buildDir = KFiles.makeDir(project.directory, project.buildDirectory!!)
         val outputDir = KFiles.makeDir(buildDir.path, "libs")
-        val mavenId = MavenId.create(project.group!!, project.artifactId!!, project.packaging, project.version!!)
+        val mavenId = MavenId(project.group!!, project.artifactId!!, project.version!!, project.packaging)
```

Comment:
This order is better for manual creation because of default arguments: it is better to have optional arguments at the end